tensorboard/plugins/image/summary_v2.py

```diff
@@ -27,6 +27,41 @@
 def image(name, data, step=None, max_outputs=3, description=None):
     """Write an image summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
+
+    Logs a set of images to the current log directory for later analysis
```

Comment:
I'd say "Writes a collection of images to the current default summary writer..." here:

- "writes" vs "logs" for consistency
- "collection" vs "set" because set has a mathematical meaning (uniqueness) that's not relevant here. ditto for usage of "set" below
- while we only have a file writer / logdir writer right now, the API is a bit more abstract than that, so better to say writer vs log directory

Reply:
Done, all.